### DISCUSSION TITLE
Future plans

### DISCUSSION THREAD

**Discussion by remcohh** [motor-admin/motor-admin#49] [General]
Hi, it seems to me that this project has the potential of becoming more than an admin panel. I think it could be the base of a lowcode development platform. Is this also your intention?

**Comment by omohokcoj** [comment]
@remcohh thanks for the feedback! that's true - motor admin can easily be converted into a lowcode dev platform given the set of features it already has. 
The only question is whether it worth to invest time in it when there are already good products like Supabase which are more robust/established in the lowcode domain.
Motor Admin is an open-source software but also is trying to make $ selling Pro features - so from this POV it makes sense to focus on a smaller product instead of trying to do 'everything'.
That's why Motor Admin positions itself as an admin dashboard tool for Supabase https://www.getmotoradmin.com/supabase - instead of trying to replicate Supabase features.
